scania-digital-design-system/tegel
# fix: rename variant option none to default

**Describe pull-request**  
Rename variant option 'none' to 'default'

**Solving issue**  
Fixes: [DTS-2166](https://tegel.atlassian.net/browse/DTS-2166)

**How to test**  
1. Go to PR preview link
2. Check if prop variant has "default" options instead of 'none'

**Suggested test steps**


[DTS-2166]: https://tegel.atlassian.net/browse/DTS-2166?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
